src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html

```diff
@@ -1,60 +1,57 @@
-<div class="modal-header">
-  <button type="button"
-          class="close float-right"
-          aria-label="Close"
-          (click)="modalRef.hide()">
-    <span aria-hidden="true">&times;</span>
-  </button>
-</div>
-<div class="modal-body">
-  <h2>
-    <img src="assets/Ceph_Logo_Stacked_RGB_120411_fa_348x348.png"
-         class="ceph-logo"
-         alt="{{ projectConstants.organization }}">
-    {{ projectConstants.organization }}
-  </h2>
-  <h3>
-    <strong>{{ projectConstants.projectName }}</strong>
-  </h3>
-  <div class="product-versions">
-    <strong>Version</strong>
-    <br>
-    {{ versionNumber }}
-    {{ versionHash }}
+<div class="about-container">
+  <div class="modal-header">
+    <button type="button" class="close float-right" aria-label="Close" (click)="modalRef.hide()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body">
+
+    <img src="assets/Ceph_Logo_Standard_RGB_Reversed_120411_fa_cropped.png" class="ceph-logo"
+      alt="{{ projectConstants.organization }}">
```

Comment:
Really minor thing: if you keep the existing indentation/formatting, this pull review will be easier to read (it initially looked like you changed almost everything, but nothing farther from truth: it's because git-diff is not able to understand that your formatting changes are 'harmless').

Reply:
Thanks for the tip.